app/models/notification.rb

```diff
@@ -15,7 +15,7 @@ def self.for(recipient, opts={})
   end
 
   def self.notify(recipient, target, actor)
-    return nil unless target.respond_to? :notification_type
+    return nil unless target.respond_to? :notification_type || recipient != actor
```

Comment:
I don't have an account on diaspora-fr, and if I had, I doubt @Flaburgan
would let me tamper with the running pod to debug the issue.

I need a way to reproduce it on a fresh Diaspora installation.